Tools/msg/templates/zenoh/uorb_idl_header.h.em

```diff
@@ -0,0 +1,65 @@
+@{
+import genmsg.msgs
+import re
+
+from px_generate_uorb_topic_helper import * # this is in Tools/
+
+uorb_struct = '%s_s'%name_snake_case
+uorb_struct_upper = name_snake_case.upper()
+}@
+
+/****************************************************************
+
+  PX4 Cyclone DDS IDL to C Translator compatible idl struct
```

Comment:
This template could be generic and default rather than zenoh specific.

Reply:
I can restructure it, you okay with the following paths?
```
Tools/cdrstream/px_generate_cdrstream_files.py
Tools/cdrstream/templates
Tools/zenoh/px_generate_zenoh_files.py
Tools/zenoh/templates
```